exodus_lambda/functions/origin_request.py

```diff
@@ -65,7 +64,9 @@ def handler(self, event, context):
         # pylint: disable=unused-argument
 
         request = event["Records"][0]["cf"]["request"]
+        self.logger.info("request is '%s'", request)
         uri = self.resolve_aliases(request["uri"])
+        self.logger.info("uri is '%s'", uri)
```

Comment:
Could you clean up the debugging messages, please?

A log message with this info might be helpful to keep around. So, if you want, you could convert them into a more user-friendly message.